doc/homologation.md

```diff
@@ -104,9 +104,8 @@ CommandeCollectePublicationMesures v3.0
 | F305A    \* | ``lowatt-enedis subscribe 98800000000246 --idx --denomination "COGIP"``                                                |
 | F305C       | ``lowatt-enedis subscribe 25884515170669 --idx --denomination "COGIP"``                                                |
 | F300C_O1-NR | ``lowatt-enedis subscribe 25884515170669 --cdc --linky --to `date '+%Y-%m-%d' -d "+400 days"` --denomination "COGIP"`` |
-| F300C_O2-NR | ``lowatt-enedis subscribe 25884515170669 --cdc-enable --linky --denomination "COGIP"``                                 |
+| F300C_O2-NR | ``lowatt-enedis subscribe 25884515170669 --cdc-enable --linky --denomination "COGIP" --no-autorisation``                                 |
 | F300B_O2-NR | ``lowatt-enedis subscribe 98800000000246 --cdc-enable --linky --denomination "COGIP" --no-autorisation``               |
-| F300C_O3-NR | ``lowatt-enedis subscribe 25128509404458 --cdc --linky --denomination "COGIP" --no-autorisation``                      |
```

Comment:
About this F300C_O3-NR indeed I found no references in Enedis.SGE.REF.0465 version 22.1 except it was added in v20.2.

Maybe it was F300C-GK-NR which was marked as dropped in v21.5 ?

CC @cyril-L